duckduckgo/iOS
# Backmerge for 7.114.1 hotfix

See: [Do not signout user on API failure](https://github.com/duckduckgo/iOS/pull/2698)

Original PR ---
Description:

Do not signOut user on Backend Failure
Steps to test this PR:

Get a subscription, put app in background
Wait 25 minutes
Turn of Data
Open Settings
Subscription should still be visible
